dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -32,13 +32,13 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
     winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationBuilder AppNotificationBuilder::SetTimeStamp(winrt::Windows::Foundation::DateTime const& value)
     {
         auto seconds{ winrt::clock::to_time_t(value) };
-        struct tm buf;
+        struct tm buf {};
         gmtime_s(&buf, &seconds);
 
         std::wstringstream buffer;
         buffer << std::put_time(&buf, L"%FT%T");
 
-        m_timeStamp = wil::str_printf<std::wstring>(L" displayTimestamp='%wsZ'", buffer.str().c_str());
+        m_timeStamp = wil::str_printf<std::wstring>(L" displayTimestamp='%lsZ'", buffer.str().c_str());
```

Comment:
From @DrusTheAxe in a different PR: Why the trailing 'Z'? Should above be std::put_time(..., L"%FT%T%Z") ?

Reply:
The reason why it's not in std::put_time is because we want the Z to exist as the raw character in the XML to be processed later in AppNotificationBuilder.Show.